packages/flutter/lib/src/foundation/unicode.dart

```diff
@@ -76,25 +73,100 @@ class Unicode {
   /// weird order.
   ///
   /// Use [PDI] to end the isolated scope.
-  static const String FSI = '\u2068';
+  static const int FSI = 0x2068;
 
   /// `U+2069 POP DIRECTIONAL ISOLATE`
   ///
   /// End the scope of the last [LRI], [RLI], or [FSI].
-  static const String PDI = '\u2069';
+  static const int PDI = 0x2069;
 
   /// `U+200E LEFT-TO-RIGHT MARK`
   ///
   /// Left-to-right zero-width character.
-  static const String LRM = '\u200E';
+  static const int LRM = 0x200E;
 
   /// `U+200F RIGHT-TO-LEFT MARK`
   ///
   /// Right-to-left zero-width non-Arabic character.
-  static const String RLM = '\u200F';
+  static const int RLM = 0x200F;
 
   /// `U+061C ARABIC LETTER MARK`
   ///
   /// Right-to-left zero-width Arabic character.
-  static const String ALM = '\u061C';
+  static const int ALM = 0x061C;
+
+  /// `U+2026 HORIZONTAL ELLIPSIS`
+  static const int HE = 0x2026;
+
+  /// `U+FFFC Object Replacement Character`
+  static const int ORC = 0xFFFC;
+
+  /// `U+FFFD Replacement Character`
+  static const int RC = 0xFFFD;
```

Comment:
In general most abbreviations are best avoided:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations

Are these abbreviations standard somewhere? A web search for [unicode orc] doesn't seem to find anything relevant.

This file does already have a number of opaque-looking abbreviations. But for those, a search like [unicode pdi] brings up the document that makes up part of the Unicode standard and defines the Unicode Bidirectional Algorithm:
https://unicode.org/reports/tr9/
That document defines them all in section 2 (in nearly the same order as in this file), and then uses them throughout its text as it defines the details of the algorithm. So that gives them something of a case for being an exception to the general avoidance of abbreviations.

Reply:
Agree, this was an attempt at shortening them but they're not self-explanatory at all.
I couldn't find any standarised acronyms really so perhaps we'll need to go for the whole-but-slightly-shortened thing: 
```
  static const int horizontalEllipsis = 0x2026;
  static const int objReplacementChar = 0xFFFC;
  static const int replacementChar = 0xFFFD;
```